src/index.js

```diff
@@ -89,7 +89,7 @@ function config(loader, appHost, configModuleId) {
   aurelia.configModuleId = configModuleId || null;
 
   if (configModuleId) {
-    return loader.loadModule(configModuleId).then(customConfig => customConfig.configure(aurelia));
+    return loader.loadModule(configModuleId).then(customConfig => customConfig.configure && customConfig.configure(aurelia));
```

Comment:
I definitely think that throwing a descriptive error if the `configure` method is missing would be better than failing silently. Would you be willing to update this PR with that change?